src/passes/validate.rs

```diff
@@ -63,6 +63,7 @@ pub fn run(module: &Module) -> Result<()> {
                 errs: &mut errs,
                 function,
                 local,
+                module,
```

Comment:
Just wanted to call out that I only realized after the fact that this was being shared concurrently between threads!